slamon_afm/models.py

```diff
@@ -7,10 +8,24 @@
     TypeDecorator
 from sqlalchemy.orm import relationship, backref
 from sqlalchemy.orm.exc import NoResultFound
-from sqlalchemy import func
+from sqlalchemy import func, event, case, literal_column
 
 db = SQLAlchemy()
 
+from slamon_afm.stats import statsd
+
+
+class TaskException(Exception):
+    task = None
+
+    def __init__(self, *args, task=None):
```

Comment:
Actually, for some reason, it breaks the tests (It silences the linter warning tho)